.eslintrc.json

```diff
@@ -1,4 +1,6 @@
 {
+  "root": true,
```

Comment:
this is best practice so when linting it doesn't inherit any personal configs

Reply:
Good idea!